README.md

```diff
@@ -583,6 +583,27 @@ ssh -i YOUR_KEY_FILE \
 
 For a list of what is collected, see the logdog [command list](sources/logdog/src/log_request.rs).
 
+### Kdump Support
+
+Bottlerocket provides support to collect kernel crash dumps whenever the system kernel panics.
+Like other Linux distributions, Bottlerocket uses the system kernel as the crash kernel to keep the image size small.
+Unlike other distros, Bottlerocket doesn't require a separate initfamfs file to load the crash kernel.
+Once the system panics, both the dmesg log and vmcore dump are stored at `/var/log/kdump`.
+The following pages are excluded from the collected vmcore dump:
+
+* Pages filled with zero
+* Non-private cache pages
+* All cache pages
+* User process data pages
+* Free pages
+
+There area few important caveats about the provided kdump support:
+
+* The system kernel will reserve 256M for the crash kernel, only when the host has at least 2GB of memory; the reserved space won't be available for processes running in the host
+* The crash kernel will only be loaded when the `crashkernel` parameter is present in the kernel's cmdline and if there is memory reserved for it
+* For the time being, only the vmware and aws-dev variants have kdump support enabled
+* For the time being, variants built for aarch64 and that use the 5.4 kernel will use the `kexec_load` syscall instead of the newer `kexec_file_load` syscall
```

Comment:
I'd omit both of these notes; we don't talk much about the `*-dev` variants elsewhere.

We could say "Currently, only vmware variants have kdump support enabled."